### DISCUSSION TITLE
Default config?

### DISCUSSION THREAD

**Discussion by letto4135** [renovatebot/renovate#27395] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

latest

### If you're self-hosting Renovate, select which platform you are using.

Gitea or Forgejo

### What is your question?

[This](https://github.com/letto4135/renovator) repo is the setup so far. I'm wondering if there is a way that I can say that renovate.json in this repo is the default for all the repos it discovers so I don't have to put a renovate.json in every repo. I'm not sure, but is it possible that the renovate.json file in this repo is already being used as the global config and I don't realize it?

If I set `RENOVATE_CONFIG_FILE=renovate.json` it tries to look in `/github-action/renovate.json` which is even more wrong since the platform is gitea...

**Comment by rarkins** [comment]
1. Create a shared preset 
2. Configure it in onboardingConfig

**Reply by letto4135** [reply]
Onboarding config still makes a file in every repo whereas I just want the default file in the renovator repo to be used everywhere without having to duplicate the code 60 times.

**Reply by rarkins** [reply]
Put your settings in your admin config and set onboarding=false
